canonical-websites/snapcraft.io
# Replace momentjs with date-fns

Fixes https://github.com/canonical-websites/snapcraft.io/issues/1097

## QA

- Pull the branch
- `./run`
- Visit http://0.0.0.0:8004/lxd
- Open the channel map and ensure the table displays things like "1 day ago" and "2 months ago"
- Compare it to https://snapcraft.io/lxd (NOTE: some of the shorter timescales, like hours and minutes are rounded differently in the new version so may be slightly different)
- Visit http://0.0.0.0:8004/lxd/release and compare with https://snapcraft.io/lxd/release again, the rounding is slightly different.
